config/migrations/20230405153000-propagate-agenda-status-activities-to-other-graphs.sparql

```diff
@@ -0,0 +1,33 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX generiek: <http://data.vlaanderen.be/ns/generiek#>
+
+INSERT {
+  GRAPH ?g {
+     ?agendaStatusActivity a ext:AgendaStatusActivity ;
```

Comment:
Hier ook, kan je `prov:Activity` ook toevoegen?

Reply:
deze migratie om andere graphs aan te passen doen we beter niet.
Daarvoor dient Yggdrasil rebuild.